core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3672,6 +3672,7 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
         HANDLE_OP(WASM_OP_REF_FUNC)
 #endif
 #if WASM_ENABLE_EXCE_HANDLING == 0
+        /* if exception handling is disabled, these opcodes issue a trap */
```

Comment:
Should also add `HANDLE_OP(EXT_OP_TRY)`.

Reply:
done